src/services/zendesk/filterTicketsByKeyword.ts

```diff
@@ -0,0 +1,50 @@
+import getNewTickets from "./getNewTicketsService";
+import ticket from "../../models/ticketModel";
+require('dotenv').config();
+
+function checker(subject:string,keywords:string[]) {
+    
+    for (let i:number = 0; i < keywords.length; i++) {
+      if (subject.indexOf(keywords[i]) > -1) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+const filterTicketsByKeyword = async () => {
+    let newTickets:ticket[] = await getNewTickets();
```

Comment:
new tickets should be passed as function parameter -> _const filterTicketsByKeyword = async (newTickets)_